### DISCUSSION TITLE
How do I use tag filter with UI?

### DISCUSSION THREAD

**Discussion by ycmarcoli** [fuma-nama/fumadocs#680] [Q&A] [answered]
Hi, I'm new to fuma and I would like to get some help regarding about tagging different routes under /docs, like tags for /docs/tree1 /docs/tree2 for flexsearch with fuma UI.

How do I approach this? I followed the docs and I'm getting this error.

![image](https://github.com/user-attachments/assets/69916e72-9130-4eac-a2a9-4f0c5a1983ec)

**Comment by fuma-nama** [comment]
The `page.data` is your frontmatter defined in MDX files.

The `page.data.tag` field isn't included by default, because some people may prefer using its folder as the tag instead of a custom defined tag.

you can extend the frontmatter with https://fumadocs.vercel.app/docs/mdx/validation#example on Fumadocs MDX. You may also use file path or other properties as the filter tag.

**Reply by ycmarcoli** [reply]
Sorry I didn't know I could use folders as tag. Could you guide me on how to do that? I can't seem to find it in the docs either.

**Reply by fuma-nama** [reply, marked as answer]
Just pass your desirable value to the tag field of search record. 
Like doing some calculations based on the page info: https://github.com/fuma-nama/fumadocs/blob/40d0a0a900c36d6781c0c776eca49164e0135469/apps/docs/scripts/update-index.mts#L38.
